packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -508,7 +515,7 @@ function useAutoFocus(scopeRef: RefObject<Element[] | null>, autoFocus?: boolean
     if (autoFocusRef.current) {
       activeScope = scopeRef;
       const ownerDocument = getOwnerDocument(scopeRef.current ? scopeRef.current[0] : undefined);
-      if (!isElementInScope(ownerDocument.activeElement, activeScope.current) && scopeRef.current) {
+      if (!isElementInScope(getActiveElement(ownerDocument), activeScope.current) && scopeRef.current) {
```

Comment:
@snowystinger I'm still seeing focus get lost to the body when testing "Focusable First in Scope" story with Android Talkback. Digging to see what exactly is happening but doesn't seem to get to this part of the code